v2/spanner-to-sourcedb/src/test/java/com/google/cloud/teleport/v2/templates/dbutils/dml/CassandraTypeHandlerTest.java

```diff
@@ -0,0 +1,478 @@
+        String newValuesString = "{\"values\":[\"1.1\", \"abc\", \"3.3\"]}";
+        JSONObject newValuesJson = new JSONObject(newValuesString);
+        handleFloat64ArrayType("values", newValuesJson);
+    }
+
+    @Test
+    public void testHandleDateSetType() {
+        String newValuesString = "{\"dates\":[\"2024-12-05\", \"2024-12-06\"]}";
+        JSONObject newValuesJson = new JSONObject(newValuesString);
+        Set<Date> value = handleDateSetType("dates", newValuesJson);
+
+        Set<Date> expected = new HashSet<>(Arrays.asList(
+                new GregorianCalendar(2024, Calendar.DECEMBER, 5).getTime(),
+                new GregorianCalendar(2024, Calendar.DECEMBER, 6).getTime()
+        ));
+        assertEquals(expected, value);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testHandleFloat64ArrayType_WithUnsupportedList() {
+        String jsonStr = "{\"colName\": [[1, 2, 3], [4, 5, 6]]}";
+        JSONObject valuesJson = new JSONObject(jsonStr);
+        CassandraTypeHandler.handleFloat64ArrayType("colName", valuesJson);
+    }
+
+    @Test
+    public void testHandleInt64SetType_ValidLongValues() {
+        String newValuesString = "{\"numbers\":[1, 2, 3, 4]}";
+        JSONObject newValuesJson = new JSONObject(newValuesString);
+        Set<Long> result = handleInt64SetType("numbers", newValuesJson);
+        Set<Long> expected = new HashSet<>(Arrays.asList(1L, 2L, 3L, 4L));
+        assertEquals(expected, result);
+    }
+
+    @Test
+    public void testHandleCassandraIntType_ValidInteger() {
+        String newValuesString = "{\"age\":1234}";
+        JSONObject newValuesJson = new JSONObject(newValuesString);
+        Integer result = handleCassandraIntType("age", newValuesJson);
+        Integer expected = 1234;
+        assertEquals(expected, result);
+    }
+
+    @Test
+    public void testHandleCassandraBigintType_ValidConversion() {
+        String newValuesString = "{\"age\":1234567890123}";  // A valid BigInteger value
+        JSONObject newValuesJson = new JSONObject(newValuesString);
+        Long result = handleCassandraBigintType("age", newValuesJson);
+        Long expected = 1234567890123L;
+        assertEquals(expected, result);
+    }
+
+    @Test
+    public void testByteArrayHandling_ValidByteArray() {
+        byte[] inputByteArray = {1, 2, 3, 4, 5};
+        Object colValue = inputByteArray;
+        byte[] byteArray = null;
+        if (colValue instanceof byte[]) {
+            byteArray = (byte[]) colValue;
+        }
+        assertNotNull(byteArray);
+        assertArrayEquals(inputByteArray, byteArray);
+    }
+
+    @Test
+    public void testByteArrayHandling_NotByteArray() {
+        String inputValue = "This is a string";
+        Object colValue = inputValue;
+        byte[] byteArray = null;
+        if (colValue instanceof byte[]) {
+            byteArray = (byte[]) colValue;
+        }
+        assertNull(byteArray);
+    }
+
```

Comment:
Which method is being unit tested here? Same comment for few other methods

Reply:
Address the same @shreyakhajanchi